less/_functions.less

```diff
@@ -67,22 +69,6 @@
 //}
 @plugin "plugins/color-yiq";
 
-// Retreive color Sass maps
-//@function color($key: "blue") {
-//  @return map-get($colors, $key);
-//}
-@plugin "plugins/color";
```

Comment:
While I’d really like to drop these, I’m afraid we need to keep them. 😞

Less’ map-accessors can totally do the job, but the point is to have a pretty function wrapper to simplify things. The Sass version does the same (it could have easily just used `map-get()` alone, but opted for making separate functions). Also, removing these will break existing code. The most important reason for me, though, is that if someone references the Bootstrap docs and then tries using code with these functions in them, it wouldn’t work. Like I’ve said, I’d like to keep the differences between the Less and Sass versions as minimal as possible and not having these adds a significant difference.